docs/conf.py

```diff
@@ -6,13 +6,18 @@
 else:
     import tomllib
 
+from sphinx_asdf.conf import *  # noqa: F403
+
 # The standard library importlib.metadata returns duplicate entrypoints
 # for all python versions up to and including 3.11
 # https://github.com/python/importlib_metadata/issues/410#issuecomment-1304258228
 # see PR https://github.com/asdf-format/asdf/pull/1260
 # see issue https://github.com/asdf-format/asdf/issues/1254
-from importlib_metadata import distribution
-from sphinx_asdf.conf import *  # noqa: F403
+if sys.version_info >= (3, 12):
+    from importlib.metadata import entry_points
+else:
+    from importlib_metadata import entry_points
```

Comment:
```suggestion
if sys.version_info >= (3, 12):
    from importlib.metadata import distribution
else:
    from importlib_metadata import distribution
```

Reply:
This should hopefully fix the docs failing to build.

Reply:
whoops, thanks for catching this. Done !